src/pages/careers.mdx

```diff
@@ -1,5 +1,14 @@
 import Helmet from 'react-helmet'
 import { InputButton } from 'bricks'
+import SEO from './../components/seo'
+
+<SEO
+  title="Careers | Codebrahma"
+  description="We don't offer jobs. We offer CAREERS."
+  keywords={["ruby on rails", "Ruby on Rails development", "angularjs framework", "angularjs code", "meteor js", "react js", "reactjs native", "node js development", "node js website", "spree commerce", "ruby on rails developer", "react js components", "javascript meteor", "Angular javascript", "angular framework", "js angular", "angular website", "Android Website", "ios app development", "mobile web app", "javascript android app", "web application development company"]}
+  image="https://codebrahma.com/static/40e053c59830488fb86e5427205788cf/6a612/react-colored.png"
```

Comment:
I dont think this is a right way to do it.

Reply:
added dynamic links for og:image